component/class/defaults.yml

```diff
@@ -420,6 +430,7 @@ parameters:
           helmChartVersion: ${appcat:charts:minio:version}
           grpcEndpoint: ${appcat:grpcEndpoint}
           defaultPlan: standard-1
+          sla: 99.25
```

Comment:
I've used the same values as redis/postgres, any objections?

Reply:
since we haven't got any special requirements, I'd keep it as default